python/Ganga/test/Unit/Utility/TestUtilitiesExecuteOutput.py

```diff
@@ -28,11 +28,11 @@ def test_execute_timeouts():
 
     # Test timeout doesn't delay normal command
     assert timeit.timeit(
-        '''
-        import os
-        from Ganga.Utility.execute import execute
-        execute('import os\\noutput(os.getcwd())',timeout=10, cwd=os.getcwd(), shell=False)
-        ''', number=1) < 11
+        """
+import os
+from Ganga.Utility.execute import execute
+execute('import os\\noutput(os.getcwd())',timeout=10, cwd=os.getcwd(), shell=False)
+        """, number=1) < 11
```

Comment:
Don't know why this hasn't been picked up before but the indentation was wrong for these tests.